action.yml

```diff
@@ -83,15 +91,59 @@ outputs:
 runs:
   using: composite
   steps:
+  - name: Log the next step action
+    run: echo ▷ Figuring out the environment Python version...
+    shell: bash
+  - name: Compute job Python version
+    id: compute-job-python-version
+    run: |
+      # Compute job Python version
+      import sys
+
+      # Input from GHA
+      job_python_version = '${{ inputs.job-python-version }}'
+      old_python_version = '${{ inputs.python-version }}'
+      ansible_core_version = '${{ inputs.ansible-core-version }}'
+
+      # Handle case that deprecated Python version parameter is provided
+      if old_python_version:
+          # Check whether the default of job-python-version was changed
+          if job_python_version != 'auto':
+              print(
+                  'ERROR: `python-version` and `job-python-version` cannot'
+                  ' both be supplied. Use only `job-python-version` and not'
+                  ' its deprecated alias `python-version`.'
+              )
+              sys.exit(1)
+
+          job_python_version = old_python_version
+
+      if job_python_version == 'auto':
+          # Determine maximal Python version supported by the
+          # given ansible-core version.
+          if ansible_core_version in ('stable-2.9', ):
+              job_python_version = '3.8'
+          elif ansible_core_version in ('stable-2.10', 'stable-2.11'):
+              job_python_version = '3.9'
+          elif ansible_core_version in ('stable-2.12', 'stable-2.13'):
+              job_python_version = '3.10'
+          else:
+              # For any non-stable branch, we assume the latest version
+              # supported by 'devel':
+              job_python_version = '3.10'
+
+      # Return computed job-python-version
```

Comment:
```suggestion
      # Set computed job-python-version
```

Reply:
Fixed in e1fc0e9a5d4cd43e3fa84e76aedf7aab3b8e1abc.